src/ui/sass/modules/_SearchBar.scss

```diff
@@ -280,13 +280,18 @@ $searchbar-button-text-color-active: var(--yxt-searchbar-button-text-color-base)
 
   &-buttonImage .Icon svg, .Icon-image
   {
+    display: block;
     width: 2em;
     height: 2em;
   }
 
   &-instructions {
     display: none;
   }
+
+  &-customLoadingImage {
```

Comment:
the class name on the span is capitalized, so this should be `&-CustomLoadingImage` if you're trying to target `yxt-SearchBar-CustomLoadingImage`